examples/nat_scapy_tests.py

```diff
@@ -144,9 +144,11 @@ def run_server2(serverport=12022, natport=35002):
     ackn += 1
     print "Replying with Ack"
     send(ip/TCP(sport=serverport,dport=natport,flags="A", seq=seqn,ack=ackn))
+    time.sleep(1)
```

Comment:
I'm trying to grok part of the commit message ("The second scapy test was not failing when it should.") When was it supposed to fail? Was it related to these timeouts? Did you find it straightforward to set the timeout values, or did you need to play around a bit? Thanks